x-pack/platform/plugins/shared/alerting/server/rules_client/common/filters.ts

```diff
@@ -44,20 +45,31 @@ export const buildFilter = ({
   );
 };
 
-export const buildRuleTypeIdsFilter = (ruleTypeIds?: string[]) => {
+export const buildRuleTypeIdsFilter = (ruleTypeIds?: string[], type = RULE_SAVED_OBJECT_TYPE) => {
   if (!ruleTypeIds || !ruleTypeIds?.length) {
     return;
   }
 
-  return buildFilter({ filters: ruleTypeIds, field: 'alertTypeId', operator: 'or' });
+  // why???
```

Comment:
what is the question about? 🙂

Reply:
🙂 The `alertTypeId` is a legacy terminology which, unfortunately, we cannot rename. With templates, we decided to use the correct term.